js/project.js

```diff
@@ -0,0 +1,11 @@
+document.getElementById("mobile").onclick = function() {
+    document.querySelector(".small").classList.toggle("open");
+};
```

Comment:
It's not an issue to mix Vanilla JS and jQuery, but I would follow one convention rather than two. Refactor the JS into jQuery. Should be pretty easy.